sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,14 +56,8 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit e2ee9a4720414e0a59fa9c911e9575ab564ac57c
-  # Merge: 7a5150cd be622e7c
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Sun Mar 10 18:02:50 2024 +0000
-  #     Merge pull request #1340 from Bensuo/ewan/coverity_cuda_update
-  #     [HIP][CUDA][Command-Buffer] Fix Coverity issues in HIP/CUDA command-buffer code
-  set(UNIFIED_RUNTIME_TAG e2ee9a4720414e0a59fa9c911e9575ab564ac57c)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/konradkusiak97/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG 0446c65e869d5098fa006d0035a5b00240400bee)
```

Comment:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 1cf9a08e6a98381fadbc065de2700b463d2a44a4
  # Merge: c49b116f 0446c65e
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Wed Mar 13 09:48:36 2024 +0000
  #     Merge pull request #1368 from konradkusiak97/xptiWindowsFix
  #     [CUDA][XPTI] Fix function pointer typedefs in xpti tracing
  set(UNIFIED_RUNTIME_TAG 1cf9a08e6a98381fadbc065de2700b463d2a44a4)
```

Reply:
Done

Reply:
https://github.com/intel/llvm/pull/12833 has merged now, go ahead an fix the conflict and make this ready for review, then I'll approve it.